site/content/contribute/mobile/e2e/ios.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Setup and Run iOS E2E"
+date: 2020-09-01T09:00:00-00:00
+weight: 4
+subsection: Mobile End-to-End Testing
+---
+
+#### Local setup
+1. Install [applesimutils](https://github.com/wix/AppleSimulatorUtils)
+```
+brew tap wix/brew
+brew install applesimutils
+```
+2. Set XCode's build location so that the built app, especially debug, is expected at project's location instead at the Library's folder which is unique/hashed.
```

Comment:
```suggestion
2. Set XCode's build location so that the built app, especially debug, is expected at the project's location instead of the Library's folder which is unique/hashed.
```